pom.xml

```diff
@@ -17,7 +17,7 @@
     <parent>
         <artifactId>che-parent</artifactId>
         <groupId>org.eclipse.che</groupId>
-        <version>7.0.0-beta-1.0</version>
+        <version>7.0.0-beta-2.0</version>
```

Comment:
Please, rebase on the master which is already updated to 7.0.0-beta-2.0

Reply:
done.